### ISSUE TITLE
jse-jumpbot: The bot sometimes plays the wrong record

### ISSUE THREAD

**Issue by LapplandBro** [Jump-Academy/jse#43]
The bot sometimes plays the wrong entry from the cache. [Online-Record]

Example: A player clicks on a bubble, the bot is on Easy Walls and for some reason it plays any level, but not the one the player is on.

Lazy fix: Repeat playback from the bubble - only then the recording is replayed successfully by the player's location.

In the logs - no recordings, catching the bug on video is unfortunately very rare. If it is possible with the administrators of my servers - to provoke the appearance of this flaw - the video will be provided.

**Comment by geominorai** [comment]
How recent is the build your are running? PR #36 should have addressed a possible cause behind it.

**Comment by LapplandBro** [comment]
Filename: jse_jumpbot.smx
  Title: Jump Server Essentials - JumpBot (JSE jump skills tutorial and replay bot)
  Author: AI
  Version: 1.0.0-rc5
  URL: https://jumpacademy.tf
  Status: running
  Timestamp: 08/28/2023 09:36:25
  Hash: efd7294987d701b1839fd0071c20e46a

**Comment by LapplandBro** [comment]
Oh... looks like I'm using an older version, got it.

**Comment by LapplandBro** [comment]
https://youtu.be/LdQMYk0rIK0
Build - Latest, SM 1.12, MM 1.11

**Comment by geominorai** [comment]
That's not a good example. The recordings are too close together and you are activating the playback too quickly after touching a previous bubble.

**Comment by LapplandBro** [comment]
...Maybe add some sort of 2 second timer so this doesn't happen?

Because there may be problems with `Basic Cource` on jump_academy_rc9a, since the entries there are even closer together.

**Comment by geominorai** [comment]
Yes, we will need a cooldown added anyway since some players actually have a bind or config with `+attack3` stuck on  and end up in a playback activate and cancel loop.
